core/templates/components/summary-tile/learner-topic-summary-tile.component.html

```diff
@@ -71,4 +89,49 @@
     height: 108px;
     width: 160px;
   }
+
+  /*REDESIGN CLASSES*/
+  .oppia-class-card {
+    background-color: #205d86;
+    border-radius: 0 0 .25em .25em;
+    box-shadow: 0 2px 2px 0 #00000040;
+    height: 220px;
+    margin-right: 16px;
+    padding: 0;
+    width: 210px;
+  }
+
+  .oppia-class-card:hover {
+    background-color: #00609c;
+    box-shadow: 0 0 8px 0 #00000040;
+  }
+
+  .oppia-class-card:not(:last-child) {
+    margin-right: 16px;
+  }
+
+  .oppia-class-card_button {
```

Comment:
Here and elsewhere, can we use hyphen (-) as a separator for class names?

Reply:
Done